### ISSUE TITLE
Consider a "ruby-grpc" Ubuntu package?

### ISSUE THREAD

**Issue by jgeewax** [grpc/grpc#4129]
Related to #1165 and #4128 

This would make the Ruby installation look like:

```sh
$ sudo apt-add-repository <ppa> # If we make this go away, that's bonus
$ sudo apt-get install ruby-grpc
$ ruby -e 'require "grpc"'
$
```

**Comment by jtattermusch** [comment]
@jgeewax  we just published native ruby gems (with version 0.13). Is this still an issue or can we close this?

**Comment by jgeewax** [comment]
I think this can go away.
